api/v1beta1/activemqartemis_types.go

```diff
@@ -370,6 +370,9 @@ type DeploymentPlanType struct {
 	// Specifies the Revision History Limit of the statefulset
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Revision History Limit"
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+	// Specifies the container security context
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Security Context"
+	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
```

Comment:
Adding the prefix `container` could help new users to understand that this field is related to the containers, i.e.
```suggestion
	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Container Security Context"
	ContainerSecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
```